airflow/contrib/executors/mesos_executor.py

```diff
@@ -238,15 +290,15 @@ def start(self):
 
         framework.name = get_framework_name()
 
-        if not configuration.conf.get('mesos', 'TASK_CPU'):
-            task_cpu = 1
-        else:
-            task_cpu = configuration.conf.getint('mesos', 'TASK_CPU')
+        task_cpu = configuration.conf.getint('mesos', 'TASK_CPU')
+        task_mem = configuration.conf.getint('mesos', 'TASK_MEMORY')
 
-        if not configuration.conf.get('mesos', 'TASK_MEMORY'):
-            task_memory = 256
-        else:
-            task_memory = configuration.conf.getint('mesos', 'TASK_MEMORY')
+        docker_image = None
+        # Configuration errors when DOCKER_IMAGE_SLAVE is not present in config
+        if 'DOCKER_IMAGE_SLAVE' in configuration.as_dict()['mesos']:
```

Comment:
`try: catch:` ?

Reply:
`as_dict()` on the whole airflow config object is relatively expensive, and should be avoided.

```suggestion
        if configuration.has_option('mesos', 'docker_image_slave'):
```

Additionally: Please don't use the term "slave" - "worker" or "executor" or "task" would be better terms here.